### ISSUE TITLE
signal SIGSEGV: segmentation violation

### ISSUE THREAD

**Issue by budachst** [kopia/kopia#2778]
After running fine for some days, Kopia server crashes with the above error. This looks like a memory issue to me, but I can't judge that. I have attached the output which has been captured using nohup.
[kopia_err.log.tgz](https://github.com/kopia/kopia/files/10801240/kopia_err.log.tgz)

**Comment by budachst** [comment]
This still happens on the latest version of Kopia on CentOS:

```
[root@jvmhh-archiv kopia]# rpm -q kopia
kopia-0.13.0-1.x86_64
```

Does nobody else experiences this?

**Comment by dson** [comment]
Could be the same issue that is causing https://github.com/kopia/kopia/issues/2599.

**Comment by budachst** [comment]
Yeah, that may be true. Especially, since my check of the repo also didn't reveal any corruptions. I am suspecting some memory issues, though, since this issue only appears after some days of run time of the Kopia server process.
